src/main/java/com/brownbag/exercise3/controller/AuthorController.java

```diff
@@ -0,0 +1,55 @@
+package com.brownbag.exercise3.controller;
+
+
+import com.brownbag.exercise3.entity.Author;
+import com.brownbag.exercise3.errorhandler.AuthorNotFoundException;
+import com.brownbag.exercise3.helpers.SortHelpers;
+import com.brownbag.exercise3.service.AuthorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@CrossOrigin(origins = "*")
+@RestController
+@RequestMapping("/authors")
+public class AuthorController {
+
+    @Autowired
+    private AuthorService authorService;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Author> findById(@PathVariable Integer id) throws AuthorNotFoundException {
+
+        return new ResponseEntity<Author>(authorService.findById(id), HttpStatus.OK);
+    }
+
+    @GetMapping
+    public ResponseEntity<Page<Author>> find(@RequestParam(defaultValue = "0") Integer page,
```

Comment:
Can use the exact same name in pageable (sort=id,asc&page=0&size=2), then you can use it like

  @GetMapping("/authors/{authorId}/books")
    public Page<Book> getAllBooksByAuthorId(@PathVariable (value = "authorId") Long authorId,
                                                Pageable pageable) {
        return bookRepository.findByAuthorId(authorId, pageable);
    }